src/audits/performance/speed-index-metric.js

```diff
@@ -78,7 +78,7 @@ class SpeedIndexMetric extends Audit {
 
       return {
         score: Math.round(score),
-        duration: `${results.speedIndex.toFixed(2)}ms`
+        rawValue: `${Math.round(results.speedIndex)}`
```

Comment:
does it make sense for this to be a string? If you remove the backticks it's just the rounded number.